packages/cli-repl/package.json

```diff
@@ -21,6 +21,7 @@
     "compile-ts": "tsc -p tsconfig.json",
     "start": "node bin/mongosh.js start",
     "start-async": "node bin/mongosh.js start --async",
+    "pretest": "npm run compile-ts",
```

Comment:
Should we also add `pretest-ci` then? Or is it okay because `bootstrap` will build everything anyway?

Reply:
Yeah, I think in CI we’re generally making the assumption that everything’s already built once during bootstrap and hasn’t been changed since, which seems reasonable to me :+1:

Reply:
Yep, makes sense, thanks for confirming! 😊